lib/THCUNN/SpatialFullConvolution.cu

```diff
@@ -19,8 +19,11 @@ void THNN_CudaSpatialFullConvolution_updateOutput(
   int nInputPlane = THCudaTensor_size(state, weight, 0);
   int nOutputPlane = THCudaTensor_size(state, weight, 1);
 
-  THCUNN_assertSameGPU(state, 6, input, output, weight,
-                                 bias, columns, ones);
+  THCUNN_assertSameGPU(state, 5, input, output, weight,
+                                 columns, ones);
+  if (bias) {
```

Comment:
Hey @colesbury, You are right, this looks unnecessary. My apprehension was if bias and gradbias, once set to nil, would be converted to NULL tensors or not. However since these are optionalTensors, there seems to be no problem.